src/Yasumi/Holiday.php

```diff
@@ -76,7 +78,7 @@ class Holiday extends DateTime implements JsonSerializable
     /**
      * @var string Locale (i.e. language) in which the holiday information needs to be displayed in. (Default 'en_US')
      */
-    protected $displayLocale;
```

Comment:
Please do not change the visibility of the property. Do you have any specific reason to do this?

Reply:
Not needed anymore due to usage of the `SubstituteHoliday` class.